explorer-api/src/mix_node/models.rs

```diff
@@ -98,25 +98,30 @@ pub(crate) struct NodeDescription {
     pub(crate) location: String,
 }
 
-#[derive(Serialize, Clone, Deserialize, JsonSchema)]
+#[derive(Serialize, Clone, Deserialize, JsonSchema, Debug)]
 pub(crate) struct NodeStats {
     #[serde(
         serialize_with = "humantime_serde::serialize",
         deserialize_with = "humantime_serde::deserialize"
     )]
     update_time: SystemTime,
-
     #[serde(
         serialize_with = "humantime_serde::serialize",
         deserialize_with = "humantime_serde::deserialize"
     )]
     previous_update_time: SystemTime,
 
+    #[serde(alias = "received_since_startup", default)]
```

Comment:
imo remove the `default` because it might conceal weird failures as in principle with the alias we should be able to deserialize both responses